code/modules/recycling/conveyor.dm

```diff
@@ -7,7 +7,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	base_icon_state = "conveyor"
 	name = "conveyor belt"
 	desc = "A conveyor belt."
-	layer = BELOW_OPEN_DOOR_LAYER
+	plane = FLOOR_PLANE
```

Comment:
This should probably still have a layer as well as a plane, since now its layer is undefined within the floor plane. While planes are layered, the objects inside a plane are also layered using the layer var

Reply:
objects have layer 2.9 by default which is above all the other stuff specific to the FLOOR_PLANE so it shouldn't cause issues